root/static/scripts/relationship-editor/release.js

```diff
@@ -282,15 +282,18 @@ const RE = MB.relationshipEditor = MB.relationshipEditor || {};
 
         _sortedRelationships(relationships, source) {
 
-            return relationships.filter(function (relationship) {
-                return relationship.entityTypes !== "recording-work";
+            return relationships
+                .filter(function (relationship) {
+                    return relationship.entityTypes !== "recording-work";
 
```

Comment:
While these newlines (this + line 292) were already here, I'd be happier if you could remove them while you're at it - they're pretty odd.